tests/unit/test_aggregation.py

```diff
@@ -404,16 +425,13 @@ def test_iterator__process_query_results_finished_result():
 
 
 def test_iterator__process_query_results_unexpected_result():
-    from google.cloud.datastore_v1.types import query as query_pb2
     from google.cloud.datastore.aggregation import AggregationResult
 
     iterator = _make_aggregation_iterator(None, None)
 
     aggregation_pbs = [AggregationResult(alias="total", value=1)]
 
-    more_results_enum = (
-        query_pb2.QueryResultBatch.MoreResultsType.MORE_RESULTS_TYPE_UNSPECIFIED
-    )
+    more_results_enum = 999
```

Comment:
why is this changing?

Reply:
This is related to the thread I started on the team chat the other day

The query profile response pb returns a completely empty batch object, which means the more_results field is 0, or `MORE_RESULTS_TYPE_UNSPECIFIED`.

So essentially, with query profiling, we now have to accept `MORE_RESULTS_TYPE_UNSPECIFIED ` as a valid terminal enum value (at least in this context), when before it was treated as an error

Personally I think his behavior seems to break some of the guarantees from the [API contract](https://cloud.google.com/datastore/docs/reference/data/rpc/google.datastore.v1#moreresultstype), which says the backend will never return `MORE_RESULTS_TYPE_UNSPECIFIED`. So this should probably be fixed at the backend level. But for this PR, I'm just relaxing some of the expectations on the client side